### PR TITLE
Added Call GET /activities/{id}/zones GetActivityZones(long activityId)

### PR DESCRIPTION
**Pull request by TiBall**
Added the Call to get the time spent in hartrate or power zones for an activity

### CODE REVIEW

**Comment by gabornemeth** [comment]
Thank you! 👍 
Could you add some tests for verifying successful parse of a sample answer (json sample is needed as well) of zones?

File: src/StravaSharp/ActivityClient.cs [hunk 1]
```diff
@@ -206,5 +206,19 @@ public async Task<List<Stream>> GetActivityStreams(long activityId, params Strea
             var response = await _client.RestClient.Execute<List<Stream>>(request);
             return response.Data;
         }
+
+        /// <summary>
+        /// Summit Feature. Returns the zones of a given activity. Requires activity:read for Everyone and Followers activities. Requires activity:read_all for Only Me activities.
+        /// </summary>
+        /// <param name="activityId"></param>
+        /// <returns></returns>
+        public async Task<List<ActivityZone>> GetActivitieZones(long activityId)
```
Comment (rjvdboon) [inline src/StravaSharp/ActivityClient.cs]: Nitpicker corner: method name should probably be `GetActivityZones`

Comment (gabor-nemeth) [reply src/StravaSharp/ActivityClient.cs]: @rjvdboon  Good catch!

**Comment by TiBall** [comment]
@gabornemeth : sure, I added a Test for parsing and calling the added Method GetActivityZones.
Sry for the wait. Was out riding ;-)

**Review by gabornemeth — APPROVED** [review]

**Pull request merged**